rpi0w-nexmon.sh

```diff
@@ -288,37 +259,34 @@ git clone --depth 1 https://git.kernel.org/pub/scm/linux/kernel/git/firmware/lin
 rm -rf ${basedir}/root/lib/firmware/.git
 
 # Setup build
-cd ${TOPDIR}/bcm-rpi3/
-git submodule update --recursive --remote
-source setup_env.sh
+cd ${TOPDIR}
+git clone --depth 1 https://github.com/nethunteros/re4son-raspberrypi-linux.git -b rpi-4.4.y-nexutil ${basedir}/root/usr/src/kernel
+cd ${basedir}/root/usr/src/kernel
+
 ln -s /usr/include/asm-generic /usr/include/asm
-cd ${TOPDIR}/bcm-rpi3/kernel
-git checkout rpi-4.4.y-re4son
+
+# Set default defconfig
+export ARCH=arm
+export CROSS_COMPILE=arm-linux-gnueabihf-
 
 # Set default defconfig
 make ARCH=arm CROSS_COMPILE=arm-linux-gnueabihf- bcmrpi_defconfig
 
 # Build kernel
-cd ${TOPDIR}/bcm-rpi3/firmware_patching/nexmon
-make
-cp brcmfmac/brcmfmac.ko ${basedir}/root/opt/nexmon/firmware
+make -j $(grep -c processor /proc/cpuinfo)
 
 # Make kernel modules
-cd ${TOPDIR}/bcm-rpi3/kernel/
 make modules_install INSTALL_MOD_PATH=${basedir}/root
 
 # Copy kernel to boot
-cd ${TOPDIR}/bcm-rpi3/kernel/
-perl scripts/mkknlimg --dtok ${TOPDIR}/bcm-rpi3/kernel/arch/arm/boot/zImage ${basedir}/bootp/kernel.img
-cp ${TOPDIR}/bcm-rpi3/kernel/arch/arm/boot/dts/*.dtb ${basedir}/bootp/
-cp ${TOPDIR}/bcm-rpi3/kernel/arch/arm/boot/dts/overlays/*.dtb* ${basedir}/bootp/overlays/
-cp ${TOPDIR}/bcm-rpi3/kernel/arch/arm/boot/dts/overlays/README ${basedir}/bootp/overlays/
+perl scripts/mkknlimg --dtok arch/arm/boot/zImage ${basedir}/bootp/kernel.img
+cp arch/arm/boot/dts/*.dtb ${basedir}/bootp/
+cp arch/arm/boot/dts/overlays/*.dtb* ${basedir}/bootp/overlays/
+cp arch/arm/boot/dts/overlays/README ${basedir}/bootp/overlays/
 
 # Make firmware and headers
-make ARCH=arm firmware_install INSTALL_MOD_PATH=${basedir}/root
-make ARCH=arm headers_install INSTALL_HDR_PATH=${basedir}/root/usr
-
-cp -rf ${TOPDIR}/bcm-rpi3/kernel ${basedir}/root/usr/src/kernel
+make ARCH=arm CROSS_COMPILE=arm-linux-gnueabihf- firmware_install INSTALL_MOD_PATH=${basedir}/root
+make ARCH=arm CROSS_COMPILE=arm-linux-gnueabihf- headers_install INSTALL_HDR_PATH=${basedir}/root/usr
```

Comment:
Is there a reason for the headers install?  Since the symlink will be pointing to /usr/src/kernel (where the kernel sources are), we don't actually need to install headers, and it takes up more space.